services/indexer/term/repo_term_loader_stream.js

```diff
@@ -5,22 +5,21 @@
  */
 const Transform = require("stream").Transform;
 const _         = require("lodash");
-const config    = require("../../../config");
 
 class RepoTermLoaderStream extends Transform {
 
-  constructor(termIndexer) {
+  constructor(termIndexer, config) {
     super({ objectMode: true });
     this.termIndexer = termIndexer;
     this.logger = termIndexer.logger;
-
+    this.config = config;
     this.terms = {};
-    config.TERM_TYPES_TO_INDEX.forEach((termType) => {
+    this.config.TERM_TYPES_TO_INDEX.forEach((termType) => {
```

Comment:
Super super minor, but don't need parentheses when only have one arg in an arrow function.